examples/rules/data/stories.md

```diff
@@ -15,9 +15,11 @@
 * explain                          <!-- can be anything -->
     - utter_explain_some_slot
     - loop_q_form
+    - form{"name": "loop_q_form"}
```

Comment:
in this particular case this is unnecessary, because there is this event on the top, but it doesn't change the rule if it is here, so not sure which variant is more examplarery